api/controllers/artists.py

```diff
@@ -3,8 +3,15 @@
 from sqlalchemy.orm import Session
 from api.database import get_db
 from api.schemas import ArtistSchema
-from api.models import Artist, User, PaymentProvider, PaymentUrl
-from api.schemas import ArtistBase, PaymentCreate, ArtistFull
+from api.models import (
+    Artist,
+    User,
+    PaymentProvider,
+    PaymentUrl,
+    SocialMedia,
+    SocialLink,
+)
+from api.schemas import ArtistBase, PaymentCreate, ArtistFull, SocialCreate
 from typing import Optional
 from api.utils.file_upload import upload_image
 from api.auth import get_current_user
```

Comment:
(4/4) ...and finally:
```suggestion
from api.utils.file_upload import upload_image
from api.auth import get_current_user
from sqlalchemy.orm import Session
from typing import Optional
```